spec/rubocop/cop/rspec/describe_symbol_spec.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+
+RSpec.describe RuboCop::Cop::RSpec::DescribeSymbol do
+  subject(:cop) { described_class.new }
+
+  it 'flags violations for `describe :symbol`' do
+    expect_violation(<<-RUBY)
+      describe(:some_method) { }
+               ^^^^^^^^^^^^ Avoid passing a Symbol as the first argument to `describe`.
+    RUBY
+  end
+
+  it 'flags violations for `describe :symbol` with multiple arguments' do
+    expect_violation(<<-RUBY)
+      describe(:some_method, "description") { }
+               ^^^^^^^^^^^^ Avoid passing a Symbol as the first argument to `describe`.
+    RUBY
+  end
+
+  it 'flags violations for `RSpec.describe :symbol`' do
+    expect_violation(<<-RUBY)
+      RSpec.describe(:some_method, "description") { }
```

Comment:
Might be good to have a case with a nested describe, ex:

```ruby
RSpec.describe Foo do
  describe :to_s do
           ^^^^^ Avoid passing a Symbol as the first argument to `describe`.
  end
end
```

(this case also demonstrates that `RSpec.describe` is not always flagged and a constant is accepted. maybe not totally necessary, but couldn't hurt.)